operator/internal/central/extensions/reconcile_pvc.go

```diff
@@ -26,7 +27,12 @@ const (
 	// DefaultCentralDBPVCName is the default name for Central DB PVC
 	DefaultCentralDBPVCName = "central-db"
 
+	// DefaultCentralDBBackupPVCName is the default name for Central DB backup PVC
+	DefaultCentralDBBackupPVCName = "central-db-backup"
```

Comment:
Let's either use `backup` or `backups` consistently. I'd cast a weak vote for the singular form.